openff/interchange/interop/openmm/_import/compat.py

```diff
@@ -4,6 +4,11 @@
 
 from openff.interchange.exceptions import UnsupportedImportError
 
+_UNSUPPORTED_VIRTUAL_SITE_TYPES: set[type] = {
+    openmm.TwoParticleAverageSite,
+    openmm.OutOfPlaneSite,
+}
+
```

Comment:
My instinct would be to flip this and explicitly list the supported virtual site types, so that if OpenMM adds a new virtual site type this won't need to be updated.

If there's a nice place/name for it, making this constant public would allow the supported virtual sites to be documented automatically directly from the code.